wikimedia/wikipedia-ios
# Diff user menu action change

**Phabricator:** 
https://phabricator.wikimedia.org/T341896

### Notes
Another tweak request from analytics.

### Test Steps
1. Confirm https://gerrit.wikimedia.org/r/c/schemas/event/secondary/+/945628 is merged first, then open a diff view. Tap on the user names and confirm they POST with the appropriate `diff_user_menu_previous` and `diff_user_menu_display` names.
